package.json

```diff
@@ -99,6 +102,7 @@
 		"@hendt/xml2json": "^1.0.3",
 		"@hpi-schul-cloud/commons": "^1.3.4",
 		"@keycloak/keycloak-admin-client": "^21.0.0",
+		"@lumieducation/h5p-server": "^9.2.0",
```

Comment:
This is the packages that is mentioned in the pre-evaluation, i check it and see that only 800 weekly downloads exists, last published is one year ago and only one author exists. It is not the best base, but it look like the best that we get over npm.